Makefile

```diff
@@ -93,15 +95,16 @@ autotest:
 .PHONY: testlong
 testlong:
 	go vet ./...
-	$(GODEP) go test ./...
+	make cover
 	make -C tests test
```

Comment:
Minor comment, but wouldn't make sense to do `make cover` after the tests? Because I assume cover is slower and doesn't make sense to run it if the tests are failing.